lib/matplotlib/cbook/deprecation.py

```diff
@@ -307,7 +307,7 @@ def __repr__(self):
 _deprecated_parameter = _deprecated_parameter_class()
 
 
-def _delete_parameter(since, name, func=None):
+def _delete_parameter(since, name, func=None, **kwargs):
```

Comment:
I would explicitly mention in the docstring:

> This can also be used for parameters that are handled through `**kwargs`.

Reply:
done (just below)